test/app/invitations_test.rb

```diff
@@ -89,6 +89,14 @@ def test_invitation
     refute team.includes?(charlie)
   end
 
+  def test_team_creator_receive_no_notification
```

Comment:
I also think we should be testing this for the `Team#new` route. It looks like they take somewhat different code paths... [new invitations](https://github.com/exercism/exercism.io/blob/master/app/routes/invitations.rb#L7) vs [new team](https://github.com/exercism/exercism.io/blob/master/app/routes/teams.rb#L34).